.github/workflows/image-go.yml

```
@@ -162,7 +162,7 @@ jobs:
         git add .
         if git status | grep -q "Changes to be committed"
         then
-          git commit --message "Update dev image"
+          git commit --message "Update dev image ${{ github.event.repository.name }}"
```

You are missing any repository suffix that might exist, so this could be a misleading message.
How about altering the previous step to stash information in env var. Something like (untested):
```yaml
    - name: Update image tag
      if: inputs.push-image
      run: |
        cd apps-infrastructure
        echo 'REPO_IMAGE=${{ github.event.repository.name }}${{env.REPOSITORY_SUFFIX}}' >> $GITHUB_ENV
        if [[ ${{inputs.runs-as}} == "cronjob" ]]; then
          echo 'REPO_IMAGE=${{env.REPO_IMAGE}}-cronjob' >> $GITHUB_ENV
        fi
        CHART_VALUES_PATH="./charts/${{env.REPO_IMAGE}}/values/image-dev.yaml"
        printf "common:\n  image:\n    tag: \"${{env.SHORT_SHA}}\"\n" > "${CHART_VALUES_PATH}"
```
Then use the new REPO_IMAGE var:
```suggestion
          git commit --message "Update dev image ${{env.REPO_IMAGE}}"
```

---

I think it's overkill, it's an FYI, we're not relying on this to trigger something, just a slack message

---

We wanted to show exactly the source from which the commit was made, the suffix will be visible on the chart that will be updated